tools/harness-thci/OpenThread.py

```diff
@@ -261,6 +261,11 @@ def _onCommissionStart(self):
     def _onCommissionStop(self):
         """Called when commissioning stops."""
 
+    @abstractmethod
+    def __restartAgentService(self):
```

Comment:
Methods that start with `__` is private and can't be inherited. 

So, it would be pointless to use `abstractmethod` for this method?

Reply:
@simonlingoogle I'll back that out for now and et the right syntax for that.  I'm pretty new to Python, so the underscore syntax was a hidden feature until you pointed it out.  I thought that was just a naming convention!  Thanks for that.